langwatch/prisma/migrations/20260712000001_rename_public_share_to_share_link/migration.sql

```
@@ -0,0 +1,64 @@
+DROP INDEX "PublicShare_projectId_resourceType_resourceId_key";
+
+-- New columns.
+ALTER TABLE "ShareLink" ADD COLUMN "token" TEXT;
+ALTER TABLE "ShareLink" ADD COLUMN "threadId" TEXT;
+ALTER TABLE "ShareLink" ADD COLUMN "visibility" "ShareVisibility" NOT NULL DEFAULT 'PUBLIC';
+ALTER TABLE "ShareLink" ADD COLUMN "expiresAt" TIMESTAMP(3);
+ALTER TABLE "ShareLink" ADD COLUMN "maxViews" INTEGER;
+ALTER TABLE "ShareLink" ADD COLUMN "viewCount" INTEGER NOT NULL DEFAULT 0;
+
+-- Backfill token from the existing id so pre-migration links keep resolving.
+UPDATE "ShareLink" SET "token" = "id" WHERE "token" IS NULL;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)